.evergreen/ensure-binary.sh

```diff
@@ -45,49 +46,55 @@ case $_NAME in
         darwin-arm64)
           _URL="$_BASE/darwin/arm64/kubectl"
         ;;
+        *)
+           echo "Unsupported for $_NAME: $_OS_NAME-$_MARCH"
+           return 1
+        ;;
     esac
+    pushd ${DRIVERS_TOOLS}/.bin
+    "$SCRIPT_DIR/retry-with-backoff.sh" curl -L -s -O $_URL
+    "$SCRIPT_DIR/retry-with-backoff.sh" curl -L -s -O "$_URL.sha256"
+    echo "$(cat kubectl.sha256)  kubectl" | sha256sum --check
+    chmod +x kubectl
+    popd
   ;;
   gcloud)
     _BASE="https://dl.google.com/dl/cloudsdk/channels/rapid/downloads"
     case "$_OS_NAME-$_MARCH" in
        linux-x86_64)
           _URL="$_BASE/google-cloud-cli-linux-x86_64.tar.gz"
+          _SHA_SUM=1ec7d351de14277c00c430f9fb939255c4b63940417da2f24ff6413a9e221c2f
```

Comment:
Were these checksums manually computed? If they were obtained from a reference, recommend documenting that reference with a comment (e.g. above the `_BASE=...`).

Reply:
Done!